Assets/Tests/Play/DepthkitPlayTest.cs

```diff
@@ -0,0 +1,166 @@
+using UnityEngine;
+using UnityEngine.TestTools;
+using NUnit.Framework;
+using System;
+using System.Collections;
+using System.Text.RegularExpressions;
+using Vimeo.Player;
+using UnityEngine.Video;
+
+public class DepthkitPlayTest : TestConfig
+{
+#if VIMEO_DEPTHKIT_SUPPORT
+    GameObject camObj;
+    GameObject depthkitObj;
+    GameObject playerObj;
+    VimeoPlayer player;
+
+    Depthkit.Depthkit_Clip depthkitClip;
+
+    bool triggered;
+
+
+    float timeout = 10;
+    float elapsed = 0;
+    
+    [SetUp]
+    public void _Before()
+    {
+        // Camera setup
+        camObj = new GameObject("Camera");
+        camObj.AddComponent<Camera>();
+        camObj.transform.Translate(0, 0, -3);
+
+        // Player Setup
+        playerObj = new GameObject("Video Player");
+        player = playerObj.AddComponent<VimeoPlayer>();
+        player.selectedResolution = StreamingResolution.x360p;
+        player.videoPlayerType = VideoPlayerType.Depthkit;
+        player.autoPlay = false;
+
+        //Depthkit setup
+        depthkitObj = new GameObject("Depthkit Clip");
+        depthkitClip = depthkitObj.AddComponent<Depthkit.Depthkit_Clip>();
+        depthkitClip.Setup(Depthkit.AvailablePlayerType.UnityVideoPlayer, Depthkit.RenderType.Photo, new TextAsset());
+        player.depthKitClip = depthkitClip;  
+        triggered = false;
+        elapsed = 0;
+    }
+
+    [UnityTest]
+    public IEnumerator Can_Playback_Volumetric_Video_With_Valid_Token_And_Unity_VideoPlayer() 
+    {
+
+
+        player.OnVideoStart += EventTriggered;
+        
+        player.SignIn(VALID_STREAMING_TOKEN);
+        player.PlayVideo(VALID_VIMEO_VOLUMETRIC_VIDEO_ID);
+
+        while (!triggered) {
```

Comment:
I think you are implementing the old way of timing out tests, right? Should use https://github.com/vimeo/vimeo-unity-sdk/blob/master/Assets/Tests/Play/VimeoRecorderPlayTest.cs#L83